menpowidgets/base.py

```diff
@@ -673,6 +673,9 @@ def visualize_images(images, figure_size=(10, 8), style='coloured',
     browser_style : ``{'buttons', 'slider'}``, optional
         It defines whether the selector of the objects will have the form of
         plus/minus buttons or a slider.
+    custom_function: `function` or ``None``, optional
+        If not None, it should be a function that accepts an image and returns
```

Comment:
Maybe it should return a `list` of `str`, where each `str` is an additional line in the info tab?